packages/server/controllers/index.js

```diff
@@ -17,6 +17,18 @@ const transcribeHandler = async (req, res) => {
   });
 };
 
+
+const completionHandler = async (text) => {
+  // const input = "Talk me about seafish, but first tell me what day is today."
+  
+  const completion = await openAi.createChatCompletion({
+    model: "gpt-3.5-turbo",
```

Comment:
The requirement asks for GPT-4, let's store it in a constant, we'll be giving to the end user the ability to choose the model.